examples/sample-app/README.md

```diff
@@ -0,0 +1,44 @@
+# Blockly Sample App
+
+## Purpose
+
+This app illustrates how to use Blockly together with common programming tools like node/npm, webpack, typescript, eslint, and others. You can copy and modify it as much as you'd like. It contains basic infrastructure for running, building, testing, etc. that you can use even if you don't understand how to configure the related tool yet. For creating your first application using Blockly, you can use the tools we've provided here for as long as they work for you. Eventually, you might need to customize the way your application builds, for example, and at that time you can replace the provided webpack configuration with your own.
+
+## Quick Start
+
+1. [Install](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) npm if you haven't before.
+2. Clone the blockly-samples repository and navigate to this `sample-app` directory (or make a copy of it) on the command line.
+3. Run `npm install` to install the required dependencies.
+4. Run `npm run start` to run the development server and see the app in action.
+5. If you make any changes to the source code, just refresh the browser while the server is running to see them.
+
+## Tooling
+
+The application uses many of the same tools that the Blockly team uses to develop Blockly itself. Following is a brief overview, and you can read more about them on our [developer site](https://developers.google.com/blockly/guides/contribute/get-started/development_tools). 
+
+- Structure: The application is built as an npm package. You can use npm to manage the dependencies of the application.
+- Modules: ES6 modules to handle imports to/exports from other files.
+- Building/bundling: Webpack to build the source code and bundle it into one file for serving.
+- Development server: webpack-dev-server to run locally while in development.
+- Testing: Mocha to run unit tests.
+- Linting: Eslint to lint the code and ensure it conforms with a standard style.
+- UI Framework: Does not use a framework. For more complex applications, you may wish to integrate a UI framework like React or Angular.
```

Comment:
I kind of have reservations about duplicating information here, because it seems likely that these pages (readme vs devsite) will go out of sync. But I also think there's a reasonable benefit to including the info here directly.

Reply:
I would agree, but the information here is specifically about the sample app. The developer site includes information about other tools like Closure so I didn't think just pointing them there without further context / an overview made sense.